<launch>
	<node name="kinect1_tf_broadcaster" pkg="time_sync_kinects" type="tf_broadcaster.py" respawn="false">
		<param name="tf_name" type="string" value="kinect1" />
		<param name="pose" type="string" value="0 -3 2 0 0.5 1.5" />
		<param name="count" type="int" value="5" />
	</node>
	<node name="kinect2_tf_broadcaster" pkg="time_sync_kinects" type="tf_broadcaster.py" respawn="false">
                <param name="tf_name" type="string" value="kinect2" />
                <param name="pose" type="string" value="0 3 2 0 0.5 -1.5" />
                <param name="count" type="int" value="5" />
        </node>
	<node name="data_collector" pkg="time_sync_kinects" type="data_collector.py" respawn="false" output="screen">
                <param name="tf_topics" type="string" value="[kinect1, kinect2]" />
                <param name="depth_topics" type="string" value="[/camera1/depth/image_raw, /camera2/depth/image_raw]" /> 
        	<param name="save_split" type="int" value="500" />
		<param name="collection_type" type="int" value="1" />
	</node>
</launch>
